src/frontend/JasmineGraphFrontEnd.h

```diff
@@ -47,6 +48,12 @@ class JasmineGraphFrontEnd {
 
     int run();
 
+    void setServer(JasmineGraphServer s);
+
+    static void initiateEntityResolutionOrg(std::string graphID, SQLiteDBInterface sqlite, std::string masterIP, std::string designatedWorkerHost, std::string designatedWorkerPort);
```

Comment:
This line is too lengthy. Lets break into two.

Reply:
Function declaration split and length adjusted